app/code/Magento/Theme/view/frontend/templates/html/topmenu/renderer.phtml

```diff
@@ -0,0 +1,71 @@
+<?php
```

Comment:
Magento uses [alternative PHP syntax](http://php.net/manual/en/control-structures.alternative-syntax.php) for templates. This template looks inconsistent with other templates in the system.